lib/core/src/client/preview/start.js

```diff
@@ -122,24 +121,31 @@ function focusInInput(event) {
   );
 }
 
-const applyLayout = memoize(1)(layout => {
-  switch (layout) {
-    case 'centered': {
-      document.body.style =
-        'margin: 0px;display: flex;justify-content: center;align-items: center;min-height: 100vh;padding: 1rem;box-sizing: border-box;';
-      break;
-    }
-    case 'fullscreen': {
-      document.body.style = 'margin: 0;';
-      break;
-    }
-    case 'padded':
-    default: {
-      document.body.style = 'margin: 0;padding: 1rem';
-      break;
-    }
+let previousLayout;
+const applyLayout = layout => {
+  const styles = {
+    centered: `
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      min-height: 100vh;
+      margin: 0;
+      padding: 1rem;
+      box-sizing: border-box;
+    `,
+    fullscreen: `
+      margin: 0;
+    `,
+    padded: `
+      margin: 0;
+      padding: 1rem;
+    `,
+  }[layout];
+  if (styles && layout !== previousLayout) {
+    document.body.style = styles;
+    previousLayout = layout;
   }
-});
+};
```

Comment:
Why is this better over a memoized function?

Reply:
No external dependency.

Reply:
Also memoization is meant to cache the result of a function, not prevent a side-effect from running multiple times. Memoization is not the right tool for that.